app/src/androidTest/java/org/systers/mentorship/view/fragments/ChangePasswordFragmentTest.kt

```diff
@@ -140,4 +140,4 @@ class ChangePasswordFragmentTest {
         }
     }
 
-} 
+}
```

Comment:
Is this the only change @justdvnsh ?

Reply:
@isabelcosta No actually. 
the previous commit actually resolves the conflict, but yet there was one thing left so resolved that in this commit. 
What happened here was, that there were 2 issues created ,
1) to check if errors are shown when passwords do not match
2) to check if errors are shown when passwords are too weak.
Now I created PR's for both the issues. First one got merged. Now when it was this PR's turn to get merged , to caused conflict since both the PR's were modifying the same file. :):)

Reply:
😅 so is this PR supposed to be changed, or you have to add code to it @justdvnsh ?

Reply:
@isabelcosta Nah. Nothing to be changed now .

Reply:
@isabelcosta github only shows the changes made in last commit under files changed preview , if you click on view file for the same , you will be able to see the entire test added in the PR, hope this helps:)

Reply:
> @isabelcosta github only shows the changes made in last commit under files changed preview , if you click on view file for the same , you will be able to see the entire test added in the PR, hope this helps:)

@epicadk the changes are actually made can you view the file and check please as suggested avove to see the test that's added

Reply:
> > @isabelcosta github only shows the changes made in last commit under files changed preview , if you click on view file for the same , you will be able to see the entire test added in the PR, hope this helps:)
> 
> @epicadk the changes are actually made can you view the file and check please as suggested avove to see the test that's added

As far as I know github shows all the changes made in the files changed preview (unless you select a specific commit). Further if you look at the top right where it shows the amount of lines added it removed it is also showing only 1.